.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!-- 
+Thanks for opening a pull request to Hydra! We've got a few requests to help us review contributions:
+
+- Make sure that your title neatly summarizes the proposed changes.
+- Provide a short overview of the change and the value it adds.
+- Share an example to help us understand the change in user experience.
+- Please make sure your code changes are covered with tests.
+- In the case of new features or big changes, remember to adjust the documentation to reflect such features/changes.
+
+Feel free to ping committers for the review!
+-->
+
+<!-- Include an overview here -->
+
+### What's changed?
+<!-- 
+Describe in detail what you've changed.
+
+A code blurb is best. Changes to features should include an example that is executable by a new user.
+If changing documentation, a link to a preview of the page is great.
+ -->
+
+### Checklist
+<!-- These boxes may be checked after opening the pull request. -->
+
+- [ ] This pull request references any related issue by including "closes `<link to issue>`"
+	- If no issue exists and your change is not a small fix, please create an issue.
+- [ ] This pull request includes tests or only affects documentation.
+- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
```

Comment:
I don't think that we need checklist, at least right now.  again, if it becomes an issue, we should revisit.